google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ClientSideStatementValueConverters.java

```diff
@@ -494,4 +495,46 @@ public String convert(String value) {
       return value.substring(7).trim();
     }
   }
+
+  /** Converter for converting Base64 encoded string to byte[] */
+  static class ProtoDescriptorsConverter implements ClientSideStatementValueConverter<byte[]> {
+
+    public ProtoDescriptorsConverter(String allowedValues) {}
+
+    @Override
+    public Class<byte[]> getParameterClass() {
+      return byte[].class;
+    }
+
+    @Override
+    public byte[] convert(String value) {
```

Comment:
Let's also add a @NonNull annotation to the argument here. Similarly in other places wherever a null value is not expected.